src/sagemaker/clarify.py

```diff
@@ -1399,6 +1580,45 @@ def get_explainability_config(self):
         return copy.deepcopy({"shap": self.shap_config})
 
 
+class AsymmetricSHAPConfig(ExplainabilityConfig):
+    """Config class for Asymmetric SHAP algorithm for TimeSeries explainability"""
+
+    def __init__(
+        self,
+        explanation_type: str = ASYM_SHAP_DEFAULT_EXPLANATION_TYPE,
+        num_samples: Optional[int] = ASYM_SHAP_DEFAULT_NUM_SAMPLES,
+    ):
+        """Initialises config for asymmetric SHAP config.
+
+        AsymmetricSHAPConfig is used specifically and only for TimeSeries explainability purposes.
+
+        Args:
+            explanation_type (str): Type of explanation to be used
+            num_samples (None or int): Number of samples to be used in the Asymmetric SHAP
```

Comment:
Let's add that this parameter is only used for "fine_grained". It is not used for `timewise`. You can also have a validation for that

Also, you can ask for a feedback from science team to write better documentation.